src/components/inputs/TextField.js

```diff
@@ -19,12 +19,14 @@ import Text from '../text';
 import TouchableOpacity from '../touchableOpacity';
 
 const DEFAULT_COLOR_BY_STATE = {
-  default: Colors.dark40,
-  focus: Colors.blue30,
-  error: Colors.red30
+  default: Colors.grey10,
+  focus: Colors.grey10,
+  error: Colors.grey10,
+  disable: Colors.grey50
```

Comment:
'disabled' - you're missing the 'd'

Reply:
I know, I tried to follow the same format you did with `focus`(and not `focused`)

Reply:
I actually prefer disabled, I will change it